db.py

```diff
@@ -174,9 +174,9 @@ class Evaluation(db.Model):
     course_year = db.Column(db.Integer, primary_key=True)
     user_id = db.Column(db.Integer, db.ForeignKey('user.id'), primary_key=True)
     task = db.Column(db.JSON, nullable=False)
-    nbr_hours = db.Column(db.String, nullable=False)
-    workload = db.Column(db.String, nullable=False)
-    comment = db.Column(db.String)
+    nbr_hours = db.Column(db.String(10), nullable=False)
+    workload = db.Column(db.String(10), nullable=False)
+    comment = db.Column(db.String(100))
```

Comment:
```suggestion
    comment = db.Column(db.String(500))
```

500 chars is not too much for a comment zone.